respeaker_ros/scripts/respeaker_node.py

```diff
@@ -322,21 +328,38 @@ def __init__(self):
         self.pub_doa_raw = rospy.Publisher("sound_direction", Int32, queue_size=1, latch=True)
         self.pub_doa = rospy.Publisher("sound_localization", PoseStamped, queue_size=1, latch=True)
         self.pub_audio = rospy.Publisher("audio", AudioData, queue_size=10)
+        self.pub_audio_info = rospy.Publisher("audio_info", AudioInfo,
+                                              queue_size=1, latch=True)
         self.pub_speech_audio = rospy.Publisher("speech_audio", AudioData, queue_size=10)
         # init config
         self.config = None
         self.dyn_srv = Server(RespeakerConfig, self.on_config)
         # start
-        self.respeaker_audio = RespeakerAudio(self.on_audio, suppress_error=suppress_pyaudio_error)
+        self.respeaker_audio = RespeakerAudio(self.on_audio, suppress_error=suppress_pyaudio_error,
+                                              publish_multichannel=self.publish_multichannel)
+        self.n_channel = 1
+        if self.publish_multichannel:
+            self.n_channel = self.respeaker_audio.channels
```

Comment:
I am sorry for additional reviewing.
I think it would be nice to write a brief background at some positions as to why `~publish_multichannel` parameter was added (or why this node starts to support multi channel) for future developers.

Reply:
Thank you!
It's very clear and easy to understand for future developers.